evap/staff/views.py

```diff
@@ -651,10 +651,13 @@ def notify_reward_points(grantings, **_kwargs):
 
         if course.state in ['evaluated', 'reviewed'] and course.is_in_evaluation_period:
             course.reopen_evaluation()
-        if form.has_changed():
-            form.save(user=request.user)
+
         if formset.has_changed():
             formset.save()
+            # Save form, even if only formset has changed, to update last_modified_user
+            form.save(user=request.user)
+        elif form.has_changed():
+            form.save(user=request.user)
```

Comment:
remember to apply all changes resulting from https://github.com/fsr-itse/EvaP/pull/1269/files#r226777525 here as well.

Reply:
done!